# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/tests/reftest/reftest.list
# Commit: eafaeb768277
# Full Hash: eafaeb7682778bd5528293cbab3cfe121c61fc26
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2025-05-09 04:15:00
# Regressor Bug: 1961387
# File Overlap Count: 3
# Description:
#   Bug 1961387 - Simple reftest for Noto Color Emoji in vertical mode. r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D248079
# ==============================================================================

diff -r 1c1f408004e1 -r eafaeb768277 gfx/tests/reftest/reftest.list
--- a/gfx/tests/reftest/reftest.list	Thu May 08 17:42:10 2025 +0000
+++ b/gfx/tests/reftest/reftest.list	Thu May 08 17:42:11 2025 +0000
@@ -48,4 +48,5 @@
 != 1870240-colrv1-cycle.html 1870240-colrv1-cycle-notref.html
 == 1909933.html 1909933-ref.html
 fuzzy(0-2,0-6229) random-if(Android) == 1912431-emoji-globalAlpha.html 1912431-emoji-globalAlpha-ref.html
+== 1961387-1.html 1961387-1-ref.html
 skip-if(!cocoaWidget) == 1964225-1.html 1964225-1-ref.html
